pages/common/picom-trans.md

```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set opacity of current window to 90 percent:
```

Comment:
```suggestion
- Set the current window opacity to a specific percent:
```

Reply:
@flipsi I assume `current window` means `currently focused window`, right? It is a bit wordy, but I think we should at least clarify this in the first example.

Reply:
> @flipsi I assume `current window` means `currently focused window`, right? It is a bit wordy, but I think we should at least clarify this in the first example.

Correct, thanks for pointing this out! 

I [adjusted the wording](https://github.com/tldr-pages/tldr/pull/8244/commits/8f97d5e862b4b9a68ba2ebdfd48938e85fe33f7e).